src/library/Uno.Material/MaterialResourcesV2.cs

```diff
@@ -112,7 +112,7 @@ private IEnumerable<(string ResourceKey, string SharedKey, bool IsDefaultStyle)>
 			Add("MaterialOutlinedPasswordBoxStyle");
 			Add("MaterialOutlinedTextBoxStyle");
 			Add("MaterialProgressBarStyle", isImplicit: true);
-#if !WinUI
+#if !WinUI_Desktop
```

Comment:
@kazo0 please add a comment with more context like you did for the PR description please